platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -217,6 +217,13 @@ Now, any values configured in the `pom` block will take priority if present, and
 
 This fixes https://github.com/gradle/gradle/issues/12259.
 
+#### Accessors for `Settings` extensions
```

Comment:
Can we make it known that its Kotlin specific in the title instead? I don't know if my suggestion makes sense but something like this:
```suggestion
#### Accessors for `Settings` extensions in Kotlin DSL
```

Reply:
It wouldn't be a mistake, although `accessors` are Kotlin Dsl stuff essentially

Reply:
@6hundreds, it's not a great suggestion, maybe something like:
```suggestion
#### Accessors for `Settings` extensions in Kotlin build files
```